framework/src/partitioner/PetscExternalPartitioner.C

```diff
@@ -183,65 +166,151 @@ PetscExternalPartitioner::_do_partition(MeshBase & mesh, const unsigned int n_pa
     local_elem_id++;
   }
 
-  ncols = 0;
+  std::vector<dof_id_type> partition;
+  // Partition graph
+  partitionGraph(comm(),
```

Comment:
Good thing this comment is here otherwise I wouldn't know we were about to do a partition graph 😆

Reply:
Hahaha. Great comment